.github/workflows/build.yml

```diff
@@ -29,10 +31,25 @@ jobs:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
 
       - name: Opam dependencies
-        run: opam install --deps-only -t .
+        run: opam install --deps-only -t ./ocamlformat.opam
```

Comment:
What was wrong with this? The new command misbehave but I can't explain why.

Reply:
I tried this because it was this way in the test-branch action, just trying a few things to make it work.

Reply:
The previous action managed to build ocamlformat, so I think this shouldn't be changed.